src/tracking/LegacyEventTracking/mapNotShownEvent.ts

```diff
@@ -1,21 +1,37 @@
-import { BannerNotShownReasons } from '@src/page/BannerNotShownReasons';
+import { BannerNotShownReasons, isNotShownReason } from '@src/page/BannerNotShownReasons';
 import { TrackingEvent } from '@src/tracking/TrackingEvent';
 import { WMDESizeIssueEvent } from '@src/tracking/WPORG/WMDEBannerSizeIssue';
+import { WMDELegacyBannerEvent } from '@src/tracking/WPORG/WMDELegacyBannerEvent';
+import { NotShownCustomData } from '@src/tracking/events/NotShownEvent';
 
-const notShownReasonToLegacyName = new Map<string, string>( [
+const notShownReasonToLegacyName = new Map<BannerNotShownReasons, string>( [
 	[ BannerNotShownReasons.SizeIssue, 'size_issue' ],
 	[ BannerNotShownReasons.DisallowedNamespace, 'namespace_tracking' ],
 	[ BannerNotShownReasons.UserInteraction, 'user_interaction' ]
 ] );
 
-export function mapNotShownEvent( notShownEvent: TrackingEvent ): WMDESizeIssueEvent {
-	return new WMDESizeIssueEvent(
-		notShownReasonToLegacyName.get( String( notShownEvent.customData.reason ) ),
-		{
-			viewportWidth: Number( notShownEvent.customData.viewportWidth ),
-			viewportHeight: Number( notShownEvent.customData.viewportHeight ),
-			bannerHeight: Number( notShownEvent.customData.bannerHeight )
-		},
-		1
-	);
+export const DEFAULT_UNKNOWN_EVENT = 'not-shown-for-unknown-reasons';
+
+/**
+ * @deprecated Will be removed when the new tracking schema is implemented
+ */
+export function mapNotShownEvent( notShownEvent: TrackingEvent<NotShownCustomData> ): WMDESizeIssueEvent|WMDELegacyBannerEvent {
+	if ( notShownEvent.customData.reason === BannerNotShownReasons.SizeIssue ) {
```

Comment:
We don't need this if statement anymore, and we can now always return a WMDESizeIssueEvent.

What we can do is pass it's tracking ratio like `notShownEvent.customData.reason === BannerNotShownReasons.SizeIssue ? 1 : 0`.

At some point we should work out a way to activate tracking for all 3 not shown reasons.

Reply:
I found it more logical to only track a `SizeIssueEvent` when we do want to track screen sizes. I'd leave it like this for now.